Source/WebKit/GPUProcess/graphics/RemoteDisplayListRecorder.cpp

```diff
@@ -404,15 +406,18 @@ void RemoteDisplayListRecorder::drawFilteredImageBuffer(std::optional<RenderingR
     drawFilteredImageBufferInternal(sourceImageIdentifier, sourceImageRect, *cachedSVGFilter, results);
 }
 
-void RemoteDisplayListRecorder::drawGlyphs(RenderingResourceIdentifier fontIdentifier, IPC::ArrayReferenceTuple<GlyphBufferGlyph, GlyphBufferAdvance> glyphsAdvances, FloatPoint localAnchor, FontSmoothingMode fontSmoothingMode)
+void RemoteDisplayListRecorder::drawGlyphs(RenderingResourceIdentifier fontIdentifier, IPC::ArrayReferenceTuple<GlyphBufferGlyph, FloatSize> glyphsAdvances, FloatPoint localAnchor, FontSmoothingMode fontSmoothingMode)
 {
     RefPtr font = resourceCache().cachedFont(fontIdentifier);
     if (!font) {
         ASSERT_NOT_REACHED();
         return;
     }
 
-    context().drawGlyphs(*font, Vector(glyphsAdvances.span<0>()), Vector(glyphsAdvances.span<(1)>()), localAnchor, fontSmoothingMode);
+    auto advances = Vector<GlyphBufferAdvance>(glyphsAdvances.span<1>().size(), [&](size_t i) {
```

Comment:
This code has become less clear.  Same in RemoteRenderingBackend::cacheDecomposedGlyphs.  Can we construct the Vector without a lambda accessor?